docs/docs/guides/quickstart.md

```diff
@@ -64,4 +64,29 @@ After some initialization logs, you should see the output of the SQL query getti
 
 2. **I see an exception `java.lang.RuntimeException: Unable to retrieve any partitions with KafkaTopicsDescriptor: Topic Regex Pattern`**
 
-   This can happen if the topic configured under `STREAMS` -> `SOURCE_KAFKA_TOPIC_NAMES` in `local.properties` is new and you have not pushed any messages to it yet. Ensure that you have pushed atleast one message to the topic before you start dagger.
\ No newline at end of file
+   This can happen if the topic configured under `STREAMS` -> `SOURCE_KAFKA_TOPIC_NAMES` in `local.properties` is new and you have not pushed any messages to it yet. Ensure that you have pushed atleast one message to the topic before you start dagger.
+
+##Docker Compose Setup
```

Comment:
@sumitaich1998 The UI appears broken for this. Can you check for syntax errors ? 
<img width="1680" alt="Screenshot 2022-10-31 at 2 01 33 PM" src="https://user-images.githubusercontent.com/18068505/198965467-851219c0-412d-4ba9-a292-158c617a9fd1.png">

Reply:
For running the UI on local, you can follow this
https://github.com/odpf/dagger/tree/docker-compose/docs

Reply:
fixed